client/src/components/RIC-Widget/ProductCard.jsx

```diff
@@ -36,19 +38,22 @@ function ProductCard({ productId, index, listState, triggerDelete }) {
   }, [productId]);
 
   return (
-    <div>
-      <div className='ProductCard'>
-        <ActionButton
-          index={index}
-          listState={listState}
-          triggerDelete={triggerDelete}/>
-        <img src={image} alt={name}></img>
-        <div>{category}</div>
-        <div>{name}</div>
-        <div>${price}</div>
-        <AvgRating
-          productId={productId}/>
-      </div>
+    <div className='ProductCard'>
+      <ActionButton
+        index={index}
+        id={productId}
+        listState={listState}
+        triggerDelete={triggerDelete}
+        triggerModal={triggerModal}/>
+      <img src={image} alt={name} className='ProductImage'></img>
+      <div>{category}</div>
+      <b>{name}</b>
+        {isSale
+          ? <div>${salePrice} <strike>${price}</strike></div>
+          : <div>${price}</div>
```

Comment:
Don't have to have a boolean here for the sales price.
It can be:
salePrice
? <div>${salePrice}...</div>
: <div>${price}</div>
Also make sure to set the initial salePrice state to null.
It'll make it so there's one less hook to keep track of since salePrice won't have anything if there is no sale.

Reply:
Ohhh, neat! I actually set the salePrice to false instead and that worked :D Ty!